pkg/restapi/rp/operation/models.go

```diff
@@ -29,6 +30,7 @@ type CreateRPTenantRequest struct {
 	Scopes               []string `json:"scopes"`
 	RequiresBlindedRoute bool     `json:"requiresBlindedRoute"`
 	SupportsWACI         bool     `json:"supportsWACI"`
+	LinkedWallet         string   `json:"linkedWallet"`
```

Comment:
Should we have 'URL' suffix to make it more clear ?

Reply:
fixed